frontend/src/metabase/dashboard/hooks/use-dashboard-url-params.ts

```diff
@@ -66,50 +68,59 @@ export const useDashboardUrlParams = ({
   });
 
   useEffect(() => {
-    const hashOptions = parseHashOptions(
-      location.hash,
-    ) as DashboardUrlHashOptions;
-    setTitled(hashOptions.titled ?? titled);
-    setBordered(hashOptions.bordered ?? bordered);
-    setFont(hashOptions.font ?? font);
+    const {
+      background,
+      bordered,
+      titled,
+      font,
+      hide_parameters,
+      hide_download_button,
+    } = parseHashOptions(location.hash) as DashboardUrlHashOptions;
```

Comment:
Modify this part to make it looks like this which does pretty much almost the same thing.

https://github.com/metabase/metabase/blob/7cb5542b1caff9c5933e221903629c9583cff89a/frontend/src/metabase/public/hooks/use-embed-frame-options.ts#L16-L23